CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Braintree Android SDK Release Notes
 
+## Unreleased
+
+* PayPalNativeCheckout
+  * Bumping native-checkout version to 0.8.1
```

Comment:
Based on the `build.gradle` diff should this be `0.8.2`?

Reply:
ahh yeah thats correct let me update